pyproject.toml

```diff
@@ -12,4 +12,58 @@ exclude = ["venv"]
 "tests/test_main.py" = ["F601"]
 
 [tool.ruff.format]
-quote-style = "single"
\ No newline at end of file
+quote-style = "single"
```

Comment:
In ideal programming languages, they teach you 

" = for strings
' = for character

Reply:
Noted. Resolving once I return to my laptop (currently outside).